engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -256,6 +258,11 @@ private void initRenderGraph() {
         renderGraph.addNode(toneMappingNode, "toneMappingNode");
         renderGraph.addNode(bloomPassesNode, "bloomPassesNode");
         renderGraph.addNode(blurPassesNode, "blurPassesNode");
+        if (renderingConfig.isVrSupport()) {
+            Node copyToVRFrameBufferNode = nodeFactory.createInstance(CopyToVRFrameBuffersNode.class);
+            ((CopyToVRFrameBuffersNode) copyToVRFrameBufferNode).setOpenVRProvider(this.vrProvider);
```

Comment:
Excuse my Java inexperience here, but you mean that I should add it to the context and then pass the context into the node?